package/raphael.js

```diff
@@ -5875,7 +5905,7 @@
                 }
             }
             timestamp = params.start || anim.start || +new Date;
-            e = {
+            element.e =  e = {
```

Comment:
Yes this was a problem. Thank you for pointing this out. I was thinking hard as to why the heap memory allocation was increasing after my fix in chrome browser. Now the heap memory allocation is back to the previous value.